src/components/deviceMap/deviceMap.js

```diff
@@ -16,14 +16,18 @@ import './deviceMap.css';
 class DeviceMap extends Component {
   constructor(props) {
     super(props);
-    window.loadMap = () => MapPane.init(this.props.BingMapKey);
-    this.state = {
-      mapCallbackComplete: false
+    window.loadMap = () => {
+      MapPane.init(this.props.BingMapKey);
+      if (this.props.BingMapKey && this.props.BingMapKey !== config.STATUS_CODES.STATIC) { this.showMap(this.props); }
```

Comment:
If you are going to use `{}` place the contents on a new line. If you want to go one line, don't include the `{}`.